station-renderer-api-v0/src/main/java/net/modificationstation/stationapi/api/client/render/StationTessellator.java

```diff
@@ -37,7 +37,7 @@ default void quad(MatrixStack.Entry matrixEntry, BakedQuad quad, float[] brightn
         Vec3i vec3i = quad.getFace().vector;
         normal.set(vec3i.x, vec3i.y, vec3i.z);
         Matrix4f matrix4f = matrixEntry.getPositionMatrix();
-        normal.transform(matrixEntry.getNormalMatrix());
```

Comment:
Disabling normals completely? Why?

Reply:
This should only be a temporary thing, as the normal matrix doesn't seem to be working properly

Reply:
Coming back to this. What do you mean "temporary"? Is it a part of gui lighting fix, or something not really meant for this pull request?